src/interface/src/app/login/login.component.ts

```diff
@@ -5,17 +5,12 @@ import { Router } from '@angular/router';
 import { AuthService } from '../services';
 import { MatDialog } from '@angular/material/dialog';
```

Comment:
I don't think there's any more dialog in this component, is there? This and other references to MatDialog can probably be safely removed from this and the associated test file

Reply:
Done.